libraries/botbuilder-dialogs/src/prompts/adaptiveCardPrompt.ts

```diff
@@ -168,7 +168,7 @@ export class AdaptiveCardPrompt extends Dialog {
 
     public async beginDialog(dc: DialogContext, options: PromptOptions): Promise<DialogTurnResult> {
         // Initialize prompt state
-        const state: any = dc.activeDialog.state as PromptState;
+        const state: any = dc.activeDialog.state as AdaptiveCardPromptState;
```

Comment:
This is my fault, but can you change `any` to `AdaptiveCardPromptState`, then remove `as AdaptiveCardPromptState`